templates/c2s-gp-finder/config.yml

```diff
@@ -1,6 +1,5 @@
 name: C2S - GP Finder
 description: |
-  Connecting to Services - GP Finder (require separate loadbalancer stack or traefik lb running in the stack)
-version: 1.0.1
+  GP Finder (requires separate load balancer stack or traefik lb running in the stack)
```

Comment:
@neilbmclaughlin @adamgraves-choices is there a reason the versioning is gone? do we still need it?